kaldb/src/main/java/com/slack/kaldb/server/ArmeriaService.java

```diff
@@ -118,6 +120,15 @@ private void addTracing(ServerBuilder sb) {
                     .addScopeDecorator(ThreadContextScopeDecorator.get())
                     .build())
             .addSpanHandler(spanHandler)
+            .addSpanHandler(
+                new SpanHandler() {
+                  @Override
+                  public boolean begin(
+                      TraceContext context, MutableSpan span, TraceContext parent) {
+                    span.tag("threadName", Thread.currentThread().getName());
+                    return super.begin(context, span, parent);
```

Comment:
I believe this should return a boolean instead of invoking the super, per the documentation?

What do we think about moving this to a separate class? Will we want to expand the available common tags over time?

Reply:
Hmm! here we do `return super.begin(...)` which should return a bool?

Reply:
moved to a separate class